### PR TITLE
fix

### PR DESCRIPTION
**Pull request by hxchjm**
**Please provide issue(s) of this PR:**
Fixes # increase the ability to use code configuration，we also have another pull request for this issue .https://github.com/polarismesh/polaris-go/pull/86

**To help us figure out who should review this PR, please put an X in all the areas that this PR affects.**

- [X] Config
- [ ] Docs
- [ ] Resolver
- [ ] Performance
- [ ] Monitor and Observability 
- [X] Test and Release

**Please check any characteristics that apply to this pull request.**

- [ ] Does not have any user-facing changes. This may include API changes, behavior changes, performance improvements, etc.

### CODE REVIEW

**Comment by houseme** [comment]
Please fix ci

File: client.go [hunk 1]
```diff
@@ -22,9 +22,9 @@ import (
 	"encoding/base64"
 	"encoding/json"
 	"fmt"
-	"strings"
-
+	"github.com/polarismesh/polaris-go/pkg/config"
```
Comment (houseme) [inline client.go:25]: 这里的import是不是可以按组引入，

系统package一组，第三方package一组，项目的一组。

其他俩个文件也一起改一下，谢谢

File: client.go [hunk 2]
```diff
@@ -100,6 +102,13 @@ func WithHeaderPrefix(headerPrefix []string) DialOption {
 	})
 }
 
+// WithSrcService set the srcMetadata for dial service routing
+func WithConfig(config *config.ConfigurationImpl) DialOption {
```
Comment (houseme) [inline client.go:106]: 注释添加一下

File: global.go [hunk 3]
```diff
@@ -62,8 +64,21 @@ func PolarisContext() (api.SDKContext, error) {
...
+
+// SetPolarisConfig set the global polaris configuration
+func SetPolarisConfig(cfg config.Configuration) {
+	bs, _ := yaml.Marshal(cfg)
+	var err error
+	polarisConfig, err = config.LoadConfiguration(bs)
+	if err != nil {
+		log.Printf("load config err (%+v)", err)
+	}
+	return
```
Comment (houseme) [inline global.go:83]: return 可以删除

File: examples/quickstart/consumer/main.go [hunk 4]
```diff
@@ -38,7 +39,29 @@ func main() {
 	// grpc客户端连接获取
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
-	conn, err := grpc.DialContext(ctx, "polaris://QuickStartEchoServerGRPC/", grpc.WithTransportCredentials(insecure.NewCredentials()))
+	cfg := &config.ConfigurationImpl{
```
Comment (chuntaojun) [inline examples/quickstart/consumer/main.go:42]: 创建 Configuration 对象，应当调用 api.NewConfiguration(), 然后按实际需要进行调整，而不是手动去构造这个结构体

File: global.go [hunk 5]
```diff
@@ -62,8 +65,20 @@ func PolarisContext() (api.SDKContext, error) {
...
+	if polarisConfig == nil {
+		oncePolarisConfig.Do(func() {
+			polarisConfig = api.NewConfiguration()
+		})
+	}
 	return polarisConfig
 }
+
+// SetPolarisConfig set the global polaris configuration
+func SetPolarisConfig(cfg config.Configuration) {
```
Comment (chuntaojun) [inline global.go:77]: 这里的 marshal 也不太需要，如果传入的话，直接使用传入的左右 polarisConfig 即可

File: go.mod [hunk 6]
```diff
@@ -3,13 +3,21 @@ module github.com/polarismesh/grpc-go-polaris
 go 1.13
 
 require (
+	github.com/BurntSushi/toml v1.2.0 // indirect
```
Comment (houseme) [inline go.mod:6]: 更新indirect package 啥原因

**Comment by houseme** [comment]
CI 问题同步一下main分支

File: examples/quickstart/consumer/main.go [hunk 7]
```diff
@@ -20,13 +20,14 @@ package main
 import (
 	"context"
 	"fmt"
+	"github.com/polarismesh/polaris-go/api"
 	"log"
```
Comment (houseme) [inline examples/quickstart/consumer/main.go:24]: import 改一下

**Pull request merged**